app/components/GameButtonsPanel/GameButtonsPanel.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonGroup from '../ButtonGroup/ButtonGroup';
+
+export default function GameButtonsPanel(props) {
+  const {
+    state,
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+  } = props;
+  const {
+    pathStartingPoint,
+    pathEndingPoint,
+    startButtonPressed,
+    endButtonPressed,
+  } = state;
+  const isStartOrEndButtonDisabled = (location) => {
+    if (location.length > 0) {
```

Comment:
`location > 0` is a boolean expression, so you should shorten the function to
```js
const isStartOrEndButtonDisabled = (location) => location.length > 0
```

Reply:
nice :)

done